packages/create-sitecore-jss/src/templates/nextjs-personalize/.env

```diff
@@ -12,3 +12,6 @@ NEXT_PUBLIC_CDP_SCRIPT_URL=https://d1mj578wat5n4o.cloudfront.net/boxever-1.4.8.m
 
 # Your Sitecore CDP point of sale
 NEXT_PUBLIC_CDP_POINTOFSALE=
+
+# Timeout for CDP requests to respond within (ms)
+PERSONALIZE_MIDDLEWARE_TIMEOUT=500
```

Comment:
Yes, we might want to bump this to 10000. For local development we were seeing higher response times. 
Also, worth considering, should we introduce a `.env.development` file ([Next.js supports these](https://nextjs.org/docs/basic-features/environment-variables#default-environment-variables)) and add this here instead (so it only applies in dev)?

Reply:
done